transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/util/logs/LogMessages.kt

```diff
@@ -1,97 +1,109 @@
-    fun DETACHING_ATTACHMENT(attachmentId: String) = "Detaching attachment: $attachmentId"
-    fun ATTACHMENT_DETACHED(attachmentId: String) = "Attachment detached: $attachmentId"
-    fun SENDING_ATTACHMENT(attachmentId: String) = "Sending attachment: $attachmentId"
-    fun ATTACHMENTS_SENT(attachments: String) = "Attachments sent: $attachments"
-    fun ADD_CUSTOM_STATE(customAttributes: String, state: String) =
+    fun presigningAttachment(attachment: Attachment) = "Presigning attachment: $attachment"
+    fun uploadingAttachment(attachment: Attachment) = "Uploading attachment: $attachment"
+    fun attachmentUploaded(attachment: Attachment) = "Attachment uploaded: $attachment"
+    fun detachingAttachment(attachmentId: String) = "Detaching attachment: $attachmentId"
+    fun attachmentDetached(attachmentId: String) = "Attachment detached: $attachmentId"
+    fun sendingAttachment(attachmentId: String) = "Sending attachment: $attachmentId"
+    fun attachmentSent(attachments: Map<String, Attachment>) = "Attachments sent: $attachments"
+    fun addCustomState(customAttributes: Map<String, String>, state: String) =
         "add: $customAttributes | state = $state"
     const val ON_SENDING = "onSending"
-    fun ON_SENT_STATE(state: String) = "onSent. state = $state"
+    fun onSentState(state: String) = "onSent. state = $state"
     const val ON_ERROR = "onError"
     const val ON_MESSAGE_ERROR = "onMessageError()"
     const val ON_SESSION_CLOSED = "onSessionClosed"
-    fun MESSAGE_PREPARED_TO_SEND(message: String) = "Message prepared to send: $message"
-    fun MESSAGE_STATE_UPDATED(message: String) = "Message state updated: $message"
-    fun ATTACHMENT_STATE_UPDATED(attachment: String) = "Attachment state updated: $attachment"
-    fun MESSAGE_HISTORY_UPDATED(thisMessage: String) = "Message history updated with: $thisMessage"
+    fun messagePreparedToSend(message: Message) = "Message prepared to send: $message"
+    fun messageStateUpdated(message: Message) = "Message state updated: $message"
+    fun attachmentStateUpdated(attachment: Attachment) = "Attachment state updated: $attachment"
+    fun messageHistoryUpdated(messages: List<Message>) = "Message history updated with: $messages"
     const val CONNECT = "connect"
     const val CONNECT_AUTHENTICATED_SESSION = "connectAuthenticatedSession"
     const val DISCONNECT = "disconnect"
-    fun CONFIGURE_AUTHENTICATED_SESSION(token: String, startNew: Boolean) =
+    fun configureAuthenticatedSession(token: String, startNew: Boolean) =
         "configureAuthenticatedSession(token = $token, startNew: $startNew)"
-    fun CONFIGURE_SESSION(token: String, startNew: Boolean) =
+    fun configureSession(token: String, startNew: Boolean) =
         "configureSession (token = $token, startNew: $startNew)"
-    fun SEND_MESSAGE(text: String, customAttributes: String) =
-        "sendMessage(text = $text, customAttributes = $customAttributes)"
+    //sendmessageca**********
```

Comment:
I think we should delete theses lines

Reply:
Sorry! I had these lines to remind myself to ask a question about it.